docs/commands.rst

```diff
@@ -144,14 +144,30 @@ command line options::
     semantic-release version --patch --print
     semantic-release version --prerelease --print
 
+.. _cmd-version-option-print-tag:
+
+``--print-tag``
+***************
+
+Same as the :ref:`cmd-version-option-print` flag but prints the tag instead of the semantic version.
```

Comment:
same as below just to make sure it is differentiated from what is a tag and what is a version